src/structure/spec_model/module/forgemod/ForgeMod113.struct.ts

```diff
@@ -13,7 +13,7 @@ export class ForgeModStructure113 extends BaseForgeModStructure {
 
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     public static isForVersion(version: MinecraftVersion, libraryVersion: string): boolean {
-        return VersionUtil.isVersionAcceptable(version, [13, 14, 15, 16])
+        return VersionUtil.isVersionAcceptable(version, [13, 14, 15, 16, 17, 18, 19])
```

Comment:
```suggestion
        return VersionUtil.isVersionAcceptable(version, [13, 14, 15, 16])
```
Again, in a separate PR once 1.17+ has landed in Helios